tendrl/commons/objects/__init__.py

```diff
@@ -201,7 +202,9 @@ def load(self):
                 pass
 
         _copy = self._copy_vars()
-
+        # Check if self.value already set, use it
+        if self.value.find('{') < 0:
```

Comment:
Well you can include the rendered self.value in _copy_vars method, but dont do this self.value.find